wiki/Beatmap_Editor/Test_mode/id.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Test mode
+
+![Screenshot tampilan test mode](img/test-mode-example.jpg "Menggunakan test mode dengan autoplay")
+
+**Test mode** adalah sebuah fitur dari [beatmap editor](/wiki/Beatmap_Editor) yang mengizinkan pembuat beatmap untuk memainkan [beatmap](/wiki/Beatmap) sendiri mereka dari beatmap editor untuk dicoba. Mode ini bisi diakses melalui tombol `Test` di bawah-kanan editor.
+
+![Tombol test mode](img/test-mode-button.png "Tombol test mode di beatmap editor")
+
+*Perhatian: Memainkan sebuah beatmap di test mode telah diketahui untuk menyebabkan lag atau framedrop untuk beberapa pengguna.*
+
+Memainkan sebuah beatmap di test mode berbeda dengan memainkannya secara normal, test mode tidak akan mengirim score, tidak menampilkan papan peringkat, tidak akan membiarkan pemain gagal, dan selalu akan menampilkan informasi berikut ini di bagian kiri pada layar:
```

Comment:
uhh not sure about this one sentence here, perhaps try sth like

`Permainan di test mode memiliki beberapa perbedaan dengan permainan normal pada umumnya. Permainan di test mode tidak memiliki papan peringkat, tidak terhubung dengan sistem submisi skor yang ada, tidak memungkinkan pemain untuk gagal, dan selalu akan menampilkan informasi-informasi berikut pada bagian kiri layar:`

Reply:
sure, that sounds better